apps/emqx_bridge/src/emqx_bridge_resource.erl

```diff
@@ -340,6 +340,8 @@ parse_confs(Type, Name, Conf) when ?IS_INGRESS_BRIDGE(Type) ->
 %% to hocon; keeping this as just `kafka' for backwards compatibility.
 parse_confs(<<"kafka">> = _Type, Name, Conf) ->
     Conf#{bridge_name => Name};
+parse_confs(<<"oracle">>, Name, Conf) ->
```

Comment:
I don't think this is needed for oracle, it's needed for Kafka because the Kafka connector needs to make use of it.